indico/modules/events/templates/display/administrative/event.html

```diff
@@ -0,0 +1,100 @@
+{% from 'events/display/indico/_common.html' import render_datetime %}
+{% from 'events/display/common/_manage_button.html' import render_manage_button %}
+{% from 'events/timetable/display/administrative/_common.html' import render_location %}
+
+{% macro render_event_time(event, timezone) %}
+    {% if event.start_dt.astimezone(timezone).date() == event.end_dt.astimezone(timezone).date() %}
+        {{ event.start_dt | format_date(format='full', timezone=timezone|string) }}
+        - <em class="time">{{ event.start_dt | format_time(format='medium', timezone=timezone) }}</em>
+    {% else %}
+        {% trans start_date=event.start_dt|format_date(format='full', timezone=timezone|string),
+                 start_time=event.start_dt|format_time(format='medium', timezone=timezone),
+                 end_date=event.end_dt|format_date(format='full', timezone=timezone|string),
+                 end_time=event.end_dt|format_time(format='medium', timezone=timezone) %}
+            from {{start_date}} ({{start_time}}) to {{end_date}} ({{end_time}})
+        {% endtrans %}
```

Comment:
`{% trans -%}...{%- endtrans %}` - otherwise there's awful surrounding whitespace in the extracted string